examples/debugger/indirect.c

```diff
@@ -60,8 +60,8 @@ static void notification_fn(size_t evhdlr_registration_id, pmix_status_t status,
     if (NULL != cbfunc) {
         cbfunc(PMIX_EVENT_ACTION_COMPLETE, NULL, 0, NULL, NULL, cbdata);
     }
-    ilactive = false;
     printf("\tCOMPLETE\n");
+sleep(2);
```

Comment:
Is the `sleep(2)` required here? If so can you add a comment above it as to why?